add-on/chrome/test/mochitest/browser_mozLoop_sharingListeners.js

```diff
@@ -54,6 +54,13 @@ function promiseNewTabLocation() {
     let tab = gBrowser.selectedTab;
     createdTabs.push(tab);
 
+    // Preloaded pages, e.g., about:newtab, are ready to go
+    if (tab.linkedBrowser.contentDocument.readyState === "complete") {
```

Comment:
Referencing `contentDocument` directly like this is not going to work in e10s browsers.
You can find a way to do this at https://dxr.mozilla.org/mozilla-central/source/browser/base/content/test/general/browser_tabs_close_beforeunload.js#16